data/doxygen.h

```diff
@@ -145,9 +145,11 @@ namespace gul14 {
  *
  * \subsection v2_7 Version 2.7
  *
+ * - *Released with DOOCS 22.10.0*
  * - Add repeat()
  * - Add gcd() and lcm()
  * - Add Howard Hinnant's date.h header
+ * - within_orders(0.0, 0.0, 10) results now in true
```

Comment:
I agree with going from a 2-part version number to a 3-part one (2.7 -> 2.7.1), but then IMO we should also list the changes by minor version number.